34.py

```diff
@@ -0,0 +1,31 @@
+# Поскольку разобраться в его кричалках не настолько просто, насколько легко он их 
+# придумывает, Вам стоит написать программу. Винни-Пух считает, что ритм есть, 
+# если число слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое. 
+# Фраза может состоять из одного слова, если во фразе несколько слов, то они 
+# разделяются дефисами. Фразы отделяются друг от друга пробелами. Стихотворение 
+# Винни-Пух вбивает в программу с клавиатуры. В ответе напишите “Парам пам-пам”, 
+# если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке
+# Ввод:
+# # пара-ра-рам рам-пам-папам па-ра-па-дам 
+# Вывод:
+# Парам пам-пам
+
+
+text = "пара-ра-рам рам-пам-папам па-ра-па-дам"
+vowels = ['а', 'е', 'ё', 'и', 'о', 'у', 'ы', 'э', 'ю', 'я']
```

Comment:
Тут лучше использовать просто строку.
```suggestion
vowels = 'аяуюоеёэиы'
```